rules/ioutil/def.bzl

```diff
@@ -0,0 +1,23 @@
+"""
+This Bazel extension contains the set of rule definitions to deal with generic IO.
+"""
+
+def _print_files_impl(ctx):
```

Comment:
Please comment this as well.
Is it possible to lint these files?

Reply:
Fixed with https://github.com/cloudfoundry-incubator/kubecf/pull/571/commits/73796132ab74bf44ac495e988e2cc4ce5f8d5e14.

Yes, it's possible to use the `buildifier` linter.